python-pyslurm.spec

```diff
@@ -0,0 +1,53 @@
+Name:		python-pyslurm
+Version:	20.02.0
+Release:	1%{?dist}
+Summary:	python-pyslurm
+
+Group:		Development/Libraries
+License:	GPLv2
+URL:		https://github.com/PySlurm/pyslurm
+
+BuildRequires:	slurm-devel, python, python-devel, python-pip
```

Comment:
Python2 support has reached it's end of life as of Jan 1, 2020. PySlurm should only support Python3 going forward.

Reply:
Ok, but RHEL 7 and CentOS 7 will support Python 2 until their support ends, right?

Reply:
Yes, I'm aware  of RHEL's continued support of Python 2. However, this project along with [many others](https://python3statement.org/) have decided to remove Python 2 support. With virtual environments, it's really easy to get Python 3 on your machine without interfering with the system's version of Python.

Reply:
You are right.